src/lib/api/documents.test.ts

```diff
@@ -129,6 +137,19 @@ describe("document uploads and processing", () => {
   });
 
   test.todo("documents.cancel");
+
+  test("pending", async ({ pending }) => {
+    const mockFetch = vi.fn().mockImplementation(async () => ({
+      ok: true,
+      async json() {
+        return pending;
+      },
+    }));
+
+    documents.pending(mockFetch).then((p) => {
+      expect(p).toMatchObject(pending);
+    });
+  });
```

Comment:
Should also test the error handling behavior of returning an empty array. There's an example of mocking and testing a failed fetch in [`lib/api/addons.test.ts:44-51`](https://github.com/MuckRock/documentcloud-frontend/blob/9153405b2c06aa1d6daa5efd8e2364d925bb21ed/src/lib/api/addons.test.ts#L44-L51)